### PR TITLE
Update Makefile & .travis.yml about #13

### PR DESCRIPTION
**Pull request by rafaelcn**
MY TURN: BLAME @RYUKINIX, FUCK.

Signed-off-by: Rafael Campos Nunes <rafaelnunes@engineer.com>

### CODE REVIEW

**Comment by rafaelcn** [comment]
You owe me @ryukinix hahaha :dancer:

**Comment by rafaelcn** [comment]
Now fix every wrong Makefile yourself as a penitence.

**Comment by ryukinix** [comment]
@rafaelcn I'll fix only whose I wrote the solution, like the C solutions. C++ no, thanks.

**Comment by rafaelcn** [comment]
It doesn't matter, is only a Makefile, and the C++ seems to not need the -lm `flag`, anyway, I could fix them all and attach to this commit, what do you think?

**Comment by ryukinix** [comment]
@rafaelcn I will be glad. Less cancer is fight for l better world

**Comment by rafaelcn** [comment]
Much better now, was you that created those long variables to replace a simple `.c`? I get that on large code bases this could make effect, but it is only a single file.

**Comment by rafaelcn** [comment]
I should fix the C++ makefiles as well, just a sec.

**Comment by ryukinix** [comment]
@rafaelcn Thanks.

**Comment by rafaelcn** [comment]
Is there any problem with the .cpp files name? I use `solution_01` instead of `solution_1`, should I change?

**Comment by ryukinix** [comment]
@rafaelcn Can be. Check later.

**Comment by ryukinix** [comment]
We need rebase this branch. You can do this, @rafaelcn?

**Comment by rafaelcn** [comment]
I did it already on my local branch, gonna send the updates soon.

**Comment by rafaelcn** [comment]
I gonna update `.travis` file.

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ before_install:
   - sudo apt-get install clojure -y
   - sudo apt-get install ghc -y
   - sudo apt-get install g++ gcc -y
+  - sudo apt-get install gcc-multilibs -y
```
Comment (ryukinix) [inline .travis.yml]: I think this will not works. `gcc-multlib`

Comment (rafaelcn) [inline .travis.yml]: Already fixed, I told you in Telegram.

Comment (ryukinix) [inline .travis.yml]: Great, but really... this doesn't works yet.

**Comment by ryukinix** [comment]
just to create a relationship: #13 @rafaelcn are trying to fix it with this PR.

**Comment by rafaelcn** [comment]
Nothing seems to fix this shift, I need to get in touch with the travis team.

**Comment by ryukinix** [comment]
@rafaelcn I told to you.

**Comment by rafaelcn** [comment]
Gonna send an email to them.

**Comment by ryukinix** [comment]
@rafaelcn Ok.

**Comment by ryukinix** [comment]
lain-wtf

**Comment by ryukinix** [comment]
When we can merge this, @rafaelcn? You are wating something? Like God Ilumination?

**Comment by rafaelcn** [comment]
I don't wanna fuck up the CI system, so I won't merge until it is fixed.

**Comment by ryukinix** [comment]
@rafaelcn Great.

**Comment by ryukinix** [comment]
SUUUUUUUUUUUUUUUUUUUUCKS, WHY THIS SHIT DON'T WORKS?! DIE MONSTER, YOU DON'T BELONG TO THIS WORLD!

**Pull request closed without merging**